sdk/storage/azure-storage-blob-v2/src/main/java/com/azure/v2/storage/blob/BlobServiceVersion.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.azure.v2.storage.blob;
+
+import io.clientcore.core.http.models.ServiceVersion;
+
+/**
+ * Service version of BlobClient.
+ */
+public enum BlobServiceVersion implements ServiceVersion {
+    /**
+     * Enum value 2025-01-05.
+     */
+    V2025_01_05("2025-01-05");
```

Comment:
We're going to want to support all versions we previously supported, right? How is that done in TypeSpec?